config/web_hooks.yml

```diff
@@ -1,15 +1,14 @@
 base: &default
-  perform_deliveries: <%= ENV.fetch('WEBHOOK_PERFORM_DELIVERIES', '0') == '1' %>
+  perform_deliveries: <%= ENV.fetch('WEBHOOK_PERFORM_DELIVERIES', '1') == '1' %>
```

Comment:
This is from https://github.com/3scale/porta/blob/539780953ebc6eddebcc1450a842e9d3198ea571/openshift/system/config/web_hooks.yml#L2